.github/workflows/build.yaml

```
@@ -0,0 +1,28 @@
+name: build
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  build:
+    name: Compile
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: brndnmtthws/rust-action-rustup@v1
+        with:
+          toolchain: stable
+      - uses: Swatinem/rust-cache@v2
+      - uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --release --bin retry
+        env:
+          CARGO_TARGET_DIR: "./dist/"
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: binary
+          path: dist/release/retry

```

Goes with the above suggestion to leave it at `target`:

```suggestion
          path: target/release/retry

```